examples/docs_snippets/docs_snippets_tests/integrations_tests/test_dagstermill.py

```diff
@@ -63,6 +49,7 @@ def exec_for_test(module_name, fn_name, env=None, raise_on_error=True, **kwargs)
                 cleanup_result_notebook(result)
 
 
+@pytest.mark.flaky(reruns=1)
```

Comment:
adding the flaky mark individually here so we don't have bk retry every test in the docs snippets suite